lib/model/migrations/20230104-01-add-entity-indices.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  await db.schema.table('entities', (en) => {
+    en.index('datasetId');
+  });
+
+  await db.schema.table('entity_defs', (ed) => {
+    ed.index('entityId');
+    ed.index('submissionDefId');
+  });
```

Comment:
I imagine that at some point, we'll want some index involving `current`. Here's what I think we do for `submission_defs`:

https://github.com/getodk/central-backend/blob/9d767ff84ffc8fef1fa36fd97a723139091bb72a/lib/model/migrations/20210118-01-add-current-flag-to-submission-defs.js#L11-L15

That's also something we could circle back to once we support entity versioning/editing.

Reply:
updated the index to include `current`